.kuzzlerc

```diff
@@ -10,6 +10,27 @@
   "servicesDefaultInitTimeout": 10000,
 
   /*
+   Cluster settings.
```

Comment:
Well, yes and no. The main motivation for this PR is to enable some cluster mode for Kuzzle and these settings are required for the current implementation to work.
But you're right, it can be removed from Kuzzle and injected by the cluster.